secure-a-cluster.md

```diff
@@ -6,20 +6,21 @@ expand: true
 
 Now that you have a [local cluster](start-a-local-cluster.html) up and running, let's secure it with authentication and encryption. This involves stopping the cluster, creating certificates, and restarting nodes with a few additional flags.
 
-1.  Stop the cluster:
+1.  Stop the cluster and close the Admin UI:
 
     ~~~ shell
-    $ ./cockroach quit --insecure
-    $ ./cockroach quit --insecure --http-port=8081
-    $ ./cockroach quit --insecure --http-port=8082
+    $ ./cockroach quit
+    $ ./cockroach quit --http-port=8081
+    $ ./cockroach quit --http-port=8082
     ~~~
 
     <button type="button" class="btn details collapsed" data-toggle="collapse" data-target="#details-secure1">Details</button>
-    <div id="details-secure1" class="collapse">
-        <ul>
-            <li>If you used the <code>cockroach start</code> commands on <a href="start-a-local-cluster.html">Start a Cluster</a> verbatim, the commands above will work as well. Otherwise, just set the <code>--http-port</code> flag to the ports you used.</li>
-            <li>For more details about the <code>cockroach quit</code> command, see <a href="stop-a-node.html">Stop a Node</a>.</li>
-        </ul>
+    <div id="details-secure1" class="collapse" markdown="1">
+
+    - If you used the `cockroach start` commands on [Start a Cluster](start-a-local-cluster.html) verbatim, the commands above will work as well. Otherwise, just set the `--http-port` flag to the ports you used.
+    - For more details about the `cockroach quit` command, see [Stop a Node](stop-a-node.html).
+    - If you leave the Admin UI open, when you restart the cluster with security (steps 3 and 4), you'll see "TLS handshake" errors until you adjust the URL to https (step 7).
```

Comment:
The "TLS handshake" errors you see should be fixed by https://github.com/cockroachdb/cockroach/pull/5746.